.storybook/stories/modal/modal-static.stories.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClrModal, ClrModalModule } from '@clr/angular';
+import { Parameters } from '@storybook/addons';
+import { Story } from '@storybook/angular';
+
+import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+
+const defaultStory: Story = args => ({
+  template: `
+    <style>
+        .backdrop-example-container {
+            position: relative;
+            padding: $clr_baselineRem_1;
```

Comment:
Does this work in an inline `style` tag? Neat.

Reply:
Yeah, missed that. Thanks!

Reply:
I was pretty sure it wouldn't work, lol. Thanks for fixing it.